### PR TITLE
Add keyboard brightness support

### PR DESCRIPTION
**Pull request by davidcui2**
- Updated the menu with additional 'Brightness' sub-menus
- Updated keyboard UI with Brightness slider, which update on the fly without using an update button like existing controls.
- Did not update readme with keyboard support list

Screenshot Menu Brightness
Screenshot Keyboard UI Change

**Note** I'm a React dev with little C++ knowledge from uni, please pay special attention to the C++ changes. 

I have only tested this with my Blackwidow Ultimate 2013 keyboard, with single green color back light. None of the other functionalities has any effect on my keyboard, which I kind of understand given most of the code handles chroma effects.

I have prettier installed and auto-formatted `js` files, which might be good a idea to include a shared setting at root level, which i can look into in another PR if needed.

### CODE REVIEW

File: src/main/index.js [hunk 1]
```diff
@@ -265,161 +304,213 @@ let keyboardMenu = [
...
+          addon.kbdSetModeStarlight(new Uint8Array([3, 0, 0, 0xff]));
         },
-      }
-    ]
+      },
+    ],
   },
   {
-    label: 'Set custom color',
+    label: 'Brightness',
```
Comment (davidcui2) [inline src/main/index.js:476]: Main menu change here

File: src/renderer/App.jsx [hunk 2]
```diff
@@ -1,45 +1,71 @@
...
+        deviceSelected={deviceSelected}
+        currentColor={currentColor}
+        setCurrentColor={setCurrentColor}
+      />
+      {deviceSelected == 'Mouse' && (
+        <MouseSensitivity
+          currentSensitivity={currentSensitivity}
+        ></MouseSensitivity>
+      )}
+      {deviceSelected === 'Keyboard' && (
```
Comment (davidcui2) [inline src/renderer/App.jsx:63]: Main window UI change

**Review by davidcui2 — COMMENTED** [review]
A lot of white space / trailing comma change caused by prettier

**Comment by 1kc** [comment]
This is great David! I will go over the code and test it out.
Also @davidcui2, yes please add in a global prettier setting in a new PR. I just haven't got around to doing that.

**Comment by 1kc** [comment]
It works very well. Just made a minor change to refresh the menu every time the brightness level is changed itself. (So that the value within the menu is always the latest)

**Pull request merged**